dt/uni-beta/README.md

```diff
@@ -0,0 +1,11 @@
+# Deployed Topology - Beta
+
+If you are looking for information on how to deploy the Beta based DT, then
+please the [README](../../examples/dt/uni-Beta/README.md) in the examples
```

Comment:
uni-beta, the directory name is lowecase

Reply:
Thanks, fixed.